tests/manage/test_create_storageclass_with_wrong_provisioner.py

```diff
@@ -0,0 +1,97 @@
+import pdb
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.framework.testlib import ManageTest, tier3
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier3
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_cephfs_secret.__name__,
+    create_ceph_block_pool.__name__,
+)
+class TestCreateStorageClassWithWrongProvisioner(ManageTest):
+    """
+    Create Storage Class with wrong provisioner
+    """
+    @pytest.mark.parametrize(
+        argnames="interface",
+        argvalues=[
+            pytest.param(
+                *["RBD"], marks=pytest.mark.polarion_id("OCS-620")
+            ),
+            pytest.param(
+                *["CEPHFS"], marks=pytest.mark.polarion_id("OCS-621")
+            )
+        ]
+    )
+    def test_create_storage_class_with_wrong_provisioner(self, interface):
+        """
+         Test function which creates Storage Class with
+         wrong provisioner and verifies PVC status
+        """
+        log.info(f"Creating a {interface} storage class")
+        if interface == "RBD":
+            interface_type = constants.CEPHBLOCKPOOL
+        else:
+            interface_type = constants.CEPHFILESYSTEM
+        sc_obj = helpers.create_storage_class(
+            interface_type=interface_type,
+            interface_name=self.cbp_obj.name,
+            secret_name=self.rbd_secret_obj.name,
+            provisioner="openshift.org/aws-efs"
+        )
+        log.info(
+            f"{interface}Storage class: {sc_obj.name} created successfully"
+        )
+
+        # Create PVC
+        pvc_obj = helpers.create_pvc(sc_name=sc_obj.name, wait=False)
+
+        # Check PVC status
+        pvc_output = pvc_obj.get()
+        pvc_status = pvc_output['status']['phase']
+        log.info(
+            f"Status of PVC {pvc_obj.name} after creation: {pvc_status}"
+        )
+        log.info(
+            f"Waiting for status '{constants.STATUS_PENDING}' "
+            f"for 60 seconds (it shouldn't change)"
+        )
+        pdb.set_trace()
+        try:
+            pvc_obj.ocp.wait_for_resource(
```

Comment:
why not waiting for the PVC to get to pending here with condition=constants.STATUS_PENDING?

Reply:
Done